internal/flags.go

```diff
@@ -54,7 +54,7 @@ type cfg struct {
 	Broker          string   `json:"broker" def:"tcp://localhost:1883" descr:"Local MQTT broker address"`
 	Username        string   `json:"username" descr:"Username for authorized local client"`
 	Password        string   `json:"password" descr:"Password for authorized local client"`
-	StorageLocation string   `json:"storageLocation" descr:"Location of the storage"`
+	StorageLocation string   `json:"storageLocation" def:"./" descr:"Location of the storage"`
```

Comment:
So far, by default, the storage location was an empty string and as a result the current working directory was used. With this change, we don't add the current directory as a default value in the description but we change the value itself. Some questions to answer in this case:
- is it ok to keep "./" then? "/" is OS specific path separator, most probably "." is a better option.
- did you check that the default value change doesn't change the behavior? I guess yes, but just to double check.

Just a note, in the user documentation "./" was used to emphasize that by default the current directory is used.

Reply:
I will test this with both Windows and Linux and the value "./" . If necessary, this might mean we will change it to "." .